test/algconvergence/ode_convergence_tests.jl

```diff
@@ -61,6 +61,18 @@ testTol = 0.2
     sim4 = test_convergence(dts2, prob, SIR54())
     @test sim4.𝒪est[:l2]≈4.4 atol=testTol
 
+    sim2 = test_convergence(dts3, prob, Alshina2())
+    @test sim2.𝒪est[:l2]≈1 atol=testTol
+
+    sim3 = test_convergence(dts3, prob, Alshina3())
+    @test sim3.𝒪est[:l2]≈1 atol=testTol
+
+    sim4 = test_convergence(dts3, prob, Alshina4())
+    @test sim4.𝒪est[:l2]≈3 atol=testTol
+
+    sim6 = test_convergence(dts1, prob, Alshina6())
+    @test sim6.𝒪est[:l2]≈4 atol=testTol
+
```

Comment:
The convergence tests here look weird. What we are doing here is testing whether the order of the convergence estimate is equal to the order of this method. If it doesn't equal, you can adjust ```dts``` so that it doesn't saturate, if it still doesn't converge, then there are bugs in your implementation, you need to carefully compare your implementation with this paper, probably some coefficients or the ```initialize!``` and ```perform_step!``` are wrong.

Reply:
Right, my implementation had bugs. I was so confused by FSAL being used inspite of the algorithm not being FSAL'd. 

The convergence tests make so much sense now. Thanks!